src/content/docs/en/reference/dev-toolbar-app-reference.mdx

```diff
@@ -197,104 +197,104 @@ This event can be sent from your plugin to change the state of your plugin. This
 
 ## Client-Server Communication
 
-Using [Vite's methods for client-server communication](https://vitejs.dev/guide/api-plugin.html#client-server-communication), dev overlay plugins can communicate with the server.
+Using [Vite's methods for client-server communication](https://vitejs.dev/guide/api-plugin.html#client-server-communication), Dev Toolbar Apps can communicate with the server.
 
-In addition to being able to send and receive custom messages, the dev overlay also sends the following messages, where `PLUGIN_ID` is the [plugin's ID](#id-string):
+In addition to being able to send and receive custom messages, the Dev Toolbar also sends the following messages, where `APP_ID` is the [app's ID](#id-string):
 
-### `astro-dev-overlay:PLUGIN_ID:initialized`
+### `astro-dev-toolbar:APP_ID:initialized`
 
-This message is sent when the plugin is initialized. The data for this message is empty.
+This message is sent when the app is initialized. The data for this message is empty.
 
 ```ts title="integration.ts"
 {
   // ...
   "astro:server:setup": ({ server }) => {
-    server.ws.on("astro-dev-overlay:super-plugin:initialized", () => {
-      console.log("My plugin was initialized!");
+    server.ws.on("astro-dev-toolbar:super-app:initialized", () => {
+      console.log("My app was initialized!");
     });
   },
   // ...
 }
 ```
 
-### `astro-dev-overlay:PLUGIN_ID:toggled`
+### `astro-dev-toolbar:APP_ID:toggled`
 
-This message is sent when the user clicks on the plugin icon in the UI. The data for this message is a boolean indicating whether the plugin is enabled or not.
+This message is sent when the user clicks on the app icon in the UI. The data for this message is a boolean indicating whether the app is enabled or not.
 
 ```ts title="integration.ts"
 {
   // ...
   "astro:server:setup": ({ server }) => {
-    server.ws.on("astro-dev-overlay:super-plugin:toggled", (data) => {
-      console.log(`My plugin is now ${data.state ? "enabled" : "disabled"}!`);
+    server.ws.on("astro-dev-toolbar:super-app:toggled", (data) => {
+      console.log(`My app is now ${data.state ? "enabled" : "disabled"}!`);
     });
   },
   // ...
 }
 ```
 
 :::note
-The built-in `connection` event from Vite fires **before** dev overlay plugins are initialized and therefore cannot be used directly by plugins. Instead, use the `astro-dev-overlay:PLUGIN_ID:initialized` event.
+The built-in `connection` event from Vite fires **before** Dev Toolbar apps are initialized and therefore cannot be used directly by apps. Instead, use the `astro-dev-toolbar:APP_ID:initialized` event.
 :::
 
 ## UI Toolkit
 
-The dev overlay includes a set of web components that can be used to build plugins with a consistent look and feel.
+The Dev Toolbar includes a set of web components that can be used to build apps with a consistent look and feel.
 
-### `astro-dev-overlay-window`
+### `astro-dev-toolbar-window`
 
 Shows a window with optionally a title and an icon.
```

Comment:
```suggestion
Shows a window.
```
For @Princesseuh to confirm. PR #5561 updates this content from the original and intentionally deletes this.

Reply:
Yeah, the new window component doesn't have those properties anymore, now the slot defines the content fully